kafka-client/src/main/java/dev/responsive/kafka/internal/db/LwtWriter.java

```diff
@@ -19,6 +19,7 @@
 import com.datastax.oss.driver.api.core.cql.BatchStatementBuilder;
 import com.datastax.oss.driver.api.core.cql.BatchType;
 import com.datastax.oss.driver.api.core.cql.BatchableStatement;
+import com.datastax.oss.driver.api.core.cql.BoundStatement;
```

Comment:
shall we move this and other sylla implementation types to a subpackage (like `mongo`)

Reply:
👍 yes that was my intention, but I didn't want this PR to get hard to review and touch a million files.